src/Shop.ts

```diff
@@ -358,10 +358,74 @@ export const shopDescriptions = (input: ShopUpgradeNames) => {
 
 }
 
+//strentax 07/21 Add function to convert code-name display to end-user friendly display of shop upgrades
+export const friendlyShopName = (input: ShopUpgradeNames) => {
+
+    switch (input) {
```

Comment:
this could be done with an object, ie
```ts
const names: Record<ShopUpgradeNames, string> = {
    offeringPotion: 'an obtainium potion',
   // ...
}

return names[input];